@stephengillie sorry for the ping, but I am ot sure how to proceed here! Manual installation of the manifest works for me and the installation validation error from CI is not super helpful:

```
****************************************************************************************
Installation Verification
****************************************************************************************
Status: Completed
##[error] Failed
```

Do you have an idea what could be the reason? Thank you!

--- a/manifests/k/KandoMenu/Kando/1.8.0/KandoMenu.Kando.installer.yaml
+++ b/manifests/k/KandoMenu/Kando/1.8.0/KandoMenu.Kando.installer.yaml
@@ -13,7 +13,7 @@ Dependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
 ReleaseDate: 2025-03-27
 Installers:
-- Architecture: x86
+- Architecture: x64
   InstallerUrl: https://github.com/kando-menu/kando/releases/download/v1.8.0/Kando-1.8.0.Setup.exe
   InstallerSha256: 162FAFA8347CAF2E7AF350238CE22255E80C5BCAFE80AD03252DB12369E5BBF7
 ManifestType: installer

